arex-instrumentation/redis/arex-lettuce-v5/src/main/java/io/arex/inst/lettuce/v5/cluster/RedisClusterReactiveCommandsImplWrapper.java

```diff
@@ -0,0 +1,499 @@
+        return reactiveCommandsWrapper.set(this, getRedisUri(), key, value);
+    }
+
+    @Override
+    public Mono<String> set(K key, V value, SetArgs setArgs) {
+        return reactiveCommandsWrapper.set(this, getRedisUri(), key, value, setArgs);
+    }
+
+    @Override
+    public Mono<String> setex(K key, long seconds, V value) {
+        return reactiveCommandsWrapper.setex(this, getRedisUri(), key, seconds, value);
+    }
+
+    @Override
+    public Mono<Boolean> setnx(K key, V value) {
+        return reactiveCommandsWrapper.setnx(this, getRedisUri(), key, value);
+    }
+
+    @Override
+    public Mono<Long> setrange(K key, long offset, V value) {
+        return reactiveCommandsWrapper.setrange(this, getRedisUri(), key, offset, value);
+    }
+
+    @Override
+    public Flux<V> sinter(K... keys) {
+        return reactiveCommandsWrapper.sinter(this, getRedisUri(), keys);
+    }
+
+    @Override
+    public Mono<Long> sinter(ValueStreamingChannel<V> channel, K... keys) {
+        return reactiveCommandsWrapper.sinter(this, getRedisUri(), channel, keys);
+    }
+
+    @Override
+    public Mono<V> spop(K key) {
+        return reactiveCommandsWrapper.spop(this, getRedisUri(), key);
+    }
+
+    @Override
+    public Flux<V> spop(K key, long count) {
+        return reactiveCommandsWrapper.spop(this, getRedisUri(), key, count);
+    }
+
+    @Override
+    public Mono<V> srandmember(K key) {
+        return reactiveCommandsWrapper.srandmember(this, getRedisUri(), key);
+    }
+
+    @Override
+    public Flux<V> srandmember(K key, long count) {
+        return reactiveCommandsWrapper.srandmember(this, getRedisUri(), key, count);
+    }
+
+    @Override
+    public Mono<Long> srandmember(ValueStreamingChannel<V> channel, K key, long count) {
+        return reactiveCommandsWrapper.srandmember(this, getRedisUri(), channel, key, count);
+    }
+
+    @Override
+    public Mono<Long> strlen(K key) {
+        return reactiveCommandsWrapper.strlen(this, getRedisUri(), key);
+    }
+
+    @Override
+    public Flux<V> sunion(K... keys) {
+        return reactiveCommandsWrapper.sunion(this, getRedisUri(), keys);
+    }
+
+    @Override
+    public Mono<Long> sunion(ValueStreamingChannel<V> channel, K... keys) {
+        return reactiveCommandsWrapper.sunion(this, getRedisUri(), channel, keys);
+    }
+
+    @Override
+    public Mono<Long> ttl(K key) {
+        return reactiveCommandsWrapper.ttl(this, getRedisUri(), key);
+    }
+
+    @Override
+    public Mono<String> type(K key) {
+        return reactiveCommandsWrapper.type(this, getRedisUri(), key);
+    }
+
+    @Override
+    public Mono<Long> zcard(K key) {
+        return reactiveCommandsWrapper.zcard(this, getRedisUri(), key);
+    }
+
+    @Override
+    public Mono<Long> del(K... keys) {
+        return del(Arrays.asList(keys));
+    }
+
+    @Override
+    public Mono<Long> del(Iterable<K> keys) {
+        if (ContextManager.needReplay()) {
```

Comment:
not support ignoreMock

Reply:
> not support ignoreMockI
IgnoreMock operation has been supported in io.arex.inst.redis.common.lettuce.RedisClusterReactiveResultUtil#monoReplay